data/taco.json

```diff
@@ -0,0 +1,75 @@
+{
+  "templates": [
+    "hands {user} a {quality} {type} taco filled with {meat} and topped with {topping}, {topping} and {topping}!"
+  ],
+  "parts": {
+    "type": [
+      "hard-shell",
+      "soft-shell",
+      "crispy",
+      "puffy",
+      "Indian"
+    ],
+    "quality": [
+      "spicy",
+      "mild",
+      "delicious",
+      [
+        "boring",
+        1
+      ],
+      [
+        "disgusting",
+        1
+      ],
+      "perfect"
+    ],
+    "meat": [
+      "minced beef",
+      "chicken",
+      "refried beans"
+    ],
+    "topping": [
+      [
+        "guacamole",
+        10
+      ],
+      [
+        "salsa",
+        10
+      ],
+      [
+        "sour cream",
+        10
+      ],
+      "cheese",
+      "lettuce",
+      "tomatoes",
+      "avocado",
+      "onion",
+      "scallions",
+      "jalapeños",
+      "capsicum",
+      [
+        "ghost chili",
+        1
+      ],
+      [
+        "olives",
+        1
+      ],
+      [
+        "pineapple",
+        1
+      ],
+      [
+        "raspberry",
+        1
+      ],
+      [
+        "potato",
+        1
+      ]
```

Comment:
pineapple, raspberry, potato? what the fuck are these @DSPhat???? HUH? WHAT ARE THESE?!?!? WHY HAVE YOU FORSAKEN THE TACO!??@!??!$@?@!?%@!